vehicle_config:
  length: 2.4
  hight: 1.74
  width: 0.97
  wheel_base: 1.67
  wheel_angle_max: 33.747     # [degrees]
  steering_angle_max: 442.208 # [degrees]
  steering_offset: 1024       # 0x400
  brake_offset: 22